pkg/workflows/testdata/fixtures/workflows/references/passing_1.yaml

```diff
@@ -7,11 +7,13 @@ triggers:
 consensus:
   - id: offchain_reporting@1.0.0
     ref: offchain_reporting_1
+    inputs: $(trigger.outputs)
     config: {}
 
 targets:
   - id: write_polygon_mainnet@1.0.0
     ref: write_polygon_mainnet_1 
+    inputs: $(offchain_reporting_1.outputs)
```

Comment:
@bolekk This tests the "flat" case

Reply:
And the non-flat case is tested elsewhere (in a file not covered by this PR since that's pre-existing behaviour)